recipes/opus/all/conanfile.py

```diff
@@ -62,11 +62,11 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
 
     def package_info(self):
+        self.cpp_info.names['cmake_find_package'] = 'OPUS'
```

Comment:
```suggestion
        self.cpp_info.names['cmake_find_package'] = 'Opus'
```

Reply:
The cmake files provided by opus project are CamelCase, not UPPERCASE:

```
-- Install configuration: "Release"
-- Installing: lib/libopus.a
-- Installing: include/opus/opus.h
-- Installing: include/opus/opus_custom.h
-- Installing: include/opus/opus_defines.h
-- Installing: include/opus/opus_multistream.h
-- Installing: include/opus/opus_projection.h
-- Installing: include/opus/opus_types.h
-- Installing: lib/pkgconfig/opus.pc
-- Installing: lib/cmake/Opus/OpusTargets.cmake
-- Installing: lib/cmake/Opus/OpusTargets-release.cmake
-- Installing: lib/cmake/Opus/OpusConfig.cmake
-- Installing: lib/cmake/Opus/OpusConfigVersion.cmake
```

Reply:
Ok, I kept the same name but did not check it correctness. Thanks for checking